README.md

```diff
@@ -17,32 +17,40 @@ make lint
 ```
 
 ## Required ENV
-`ACCESS_TOKEN` : The POD access token used to authenticate uploads. The access tokens can be found on the `Manage Organization` page.
+`POD_ACCESS_TOKEN` : The POD access token used to authenticate uploads. The access tokens can be found on the `Manage Organization` page.
 
 `BUCKET` = The bucket containing the compressed MARCXML files to be submitted to POD.
 
 `POD_URL` =  The POD URL which includes the organization code: `https://pod.stanford.edu/organizations/{Organization Code}/uploads?stream=`
 
 `SENTRY_DSN` = If set to a valid Sentry DSN, enables Sentry exception monitoring. This is not needed for local development.
 
-`STREAM` = The POD stream to use when posting MARCXML records.
-
 `WORKSPACE` = Set to `dev` for local development, this will be set to `stage` and `prod` in those environments by Terraform.
 
-### To run locally
-NOTE: These instructions for running locally don't currently work and functionality has to be verified in our dev AWS account.
-- Build the container:
-  ```bash
-  docker build -t ppod .
-  ```
-- Run the container:
-  ```bash
-  docker run -p 9000:8080 -e WORKSPACE=dev ppod:latest
-  ```
-- Post data to the container:
+
+### Verify local changes in Dev1
+- Ensure your AWS CLI is configured with credentials for the Dev1 account.
+- Publish the lambda function:
   ```bash
-  curl -XPOST "http://localhost:9000/2015-03-31/functions/function/invocations" -d "{}"
+  make publish-dev
+  make update-lambda-dev
   ```
-- Observe output:
-  ```
-  lambda
\ No newline at end of file
+
+#### Submit files to POD test stream
+Use the `Test` tab on the lambda to `Event JSON` that will match files in the dev1 S3 bucket:
```

Comment:
We should add a similar note here as in the alma-webhooks README, that if there are no matching files in the Dev1 bucket the Alma publishing profile will need to be rerun (and that the output here will match the number of files in the export matching the prefix used).

Reply:
Done